### ISSUE TITLE
Torrent Stacking camps

### ISSUE THREAD

**Issue by rianrain** [jeffhill/Dota2#1606]
### Description

As you might know Kunkka can stack certain camps by torrenting them so that the camp is in the air as the minute ticks over. This is a really cool feature but it's incredibly inconsistent as many camps **Cannot** be stacked like this, the worst offender being Dire ancients can be torrent stacked but Radiant can't. Basically just would like to see consistent all camps can be torrent stacked or none can. 

Of course this isn't specific to Kunkka, it's just the hero you can do this on easiest. Other examples are invoker tornado, impale and earthspike.
The cause is only certain camps boxes having a Z Axis so lifting creeps doesn't take them out of the box 
TLDR: Make all camps stackable with Torrent or make no camps stackable with Torrent

### Example Match ID (and possibly Timestamp)

na
